docs/howto/add-barman-server.md

```diff
@@ -0,0 +1,85 @@
+# Add an new Barman node to the exisiting cluster. 
+"Barman
+Allows your company to implement disaster recovery solutions for PostgreSQL databases with high requirements of business continuity.
+
+Taking an online hot backup of PostgreSQL is now as easy as ordering a good espresso coffee."  ~ https://www.pgbarman.org/
+
+## Before you begin.
+Adding a new Barman Server requires following configuration to be done in ansible .
+
+* Add a barman user in the `environments/<env>/vault.yml`
+```
+POSTGRES_USERS:
```

Comment:
I think this is lowercase `postgres_users`

Reply:
`POSTGRES_USERS` is in the uppercase.